### ISSUE TITLE
Readme for the examples

### ISSUE THREAD

**Issue by raviabhiram** [pyg-team/pytorch_geometric#950]
## 🚀 Feature

Would it be possible to add a readme for the examples folder with a gist of what each example does?

## Motivation

I was looking to implement a graph neural network for my capstone and was looking for an idea on how to go about it. I was looking for references that I could use. I am new to neural nets and pytorch so finding similar projects from which I can learn is super important. It would just be easier if the examples had a small gist of the problem being solved.

**Comment by rusty1s** [comment]
Yes, this is in the pipeline. I understand that the examples can become a bit overwhelming and unstructured. Will work on it!

**Comment by raviabhiram** [comment]
Thanks!
